mac

```diff
@@ -157,7 +157,14 @@ brew_install_or_upgrade 'openssl'
 brew unlink openssl && brew link openssl --force
 brew_install_or_upgrade 'libyaml'
 
-ruby_version="$(curl -sSL http://ruby.thoughtbot.com/latest)"
+find_latest_ruby() {
+  curl -sSL \
+    'https://raw.githubusercontent.com/postmodern/ruby-versions/master/ruby/versions.txt' \
+    | tail -n 2 \
```

Comment:
It looks like this is grabbing the second-to-last version. Why not grab the latest one (with `tail -n1`)?